templates/generic_show_form.html.twig

```diff
@@ -629,6 +629,12 @@
                         ) }}
                      {% endif %}
 
+                     {# display is_dynamic data without input to prevent "lock field" process to be avoid #}
+                     {% if item.isField('is_dynamic') %}
+                        {% set is_dynamic_value = call('Dropdown::getYesNo', [item.fields['is_dynamic']]) %}
```

Comment:
The hook `autoinventory_information` should probably be called here.

Reply:
is already called from this file (at end) =)